readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://VovaKlimyuk.github.io/layout_moyo-header/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/218718525-3cf9368d-21d2-4369-82fb-b2c86e28c51c.png)
Should stretch to the entire width of wide screens